assembl/static2/css/components/timeline.scss

```diff
@@ -114,13 +114,26 @@
     }
   }
 
-  .txt-active {
+  .txt-active-light {
+    .timeline-link {
+      color: $first-color;
+      font-family: $font-family4;
+      &:hover {
+        font-family: $font-family
```

Comment:
;

Reply:
We should add scss-lint in pre-commit ;) But there is lots of warnings currently in our scss files.